lib/rubocop/cop/rspec/subject_stub.rb

```diff
@@ -38,6 +38,17 @@ module RSpec
       #     end
       #   end
       #
+      #   # bad
+      #   describe Article do
+      #     subject { Article }
+      #
+      #     it 'indicates that the author is unknown' do
+      #       allow_any_instance_of(subject).to receive(:author).and_return(nil)
```

Comment:
This is a consequence of usage of the above subject definition.

Do you feel that `allow_any_instance_of(Article).to receive(:author).and_return(nil)` is wrong, and why?